nfConfigApi/openapi-generator-cli-templates/api.mustache

```diff
@@ -0,0 +1,449 @@
+// SPDX-FileCopyrightText: 2025 Canonical Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+{{>partial_header}}
```

Comment:
I see you have the "partial header", would not it be better to just modify and add the `partial_header.mustache` file, along with the `README.mustache` and `api.mustache` in the `openapi-generator-cli-templates` directory because, I think, everything else is kind of standard (no changes), no? That is no need to overwrite the other files/templates. What do you think?